specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2718,6 +2769,19 @@
         }
       }
     },
+    "LinkedIntegrationRuntimeRequest": {
+      "description": "Data factory name for linked integration runtime request.",
+      "type": "object",
+      "properties": {
+        "factoryName": {
```

Comment:
@zhangyd2015 
Would it be possible to add the extension `"x-ms-client-name"` here? Currently this name is the same as that used for the URL parameter factoryName. Readability could be improved by giving this property a different generated client name.
For an example of the current generated Python client:
https://github.com/Azure/azure-sdk-for-python/pull/2865/files#diff-fd2acb15b40ac1ce8749314b7e6619c0R1041

Reply:
I think renaming will be better than using x-ms-client-name.  Maybe linkedFactoryName?

Reply:
Service code is pending deployment now and if we change the name we will need service code change and I don't want to block the deployment as we have been waiting for long time. On the other hand, there is no concept "`LinkedFactory`", and the closure class name `LinkedIntegrationRuntimeRequest` has indicate its context.

I will prefer to add the extension `x-ms-client-name` here.